plugins/outputs/opentelemetry/README.md

```diff
@@ -0,0 +1,44 @@
+# OpenTelemetry Output Plugin
+
+This plugin writes to any backend that support the OpenTelemetry Protocol (OTLP)
+
+Metrics are named by combining the metric name and field key - eg: `cpu.usage_user`
+
+Additional resource attributes can be configured by `attributes`.
+
+Additional gRPC metadata can be configured by `headers`.
```

Comment:
You might want to remove the newlines. It's just cosmetics, but I might be nicer to read. Maybe ask @sjwang90, she's good at writing documentation! :-)

Reply:
Updated the documentation.